sys-power/throttled/throttled-0.10.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+inherit python-single-r1 systemd
+
+DESCRIPTION="Workaround for Intel throttling issues in Linux."
+HOMEPAGE="https://github.com/erpalma/throttled"
+SRC_URI="https://github.com/erpalma/throttled/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
```

Comment:
I'd suggest renaming the source tarball to ${P}.gh.tar.gz to give a hint that it's retrieved from github.com (also a convention here & it's only my suggestion)